ma/maInput.cc

```diff
@@ -285,7 +285,12 @@ Input* configureIdentity(Mesh* m, SizeField* f, SolutionTransfer* s)
   in->maximumIterations = 0;
   in->shouldFixShape = false;
   in->shouldSnap = false;
-  return in;
+  return (const Input*)in;
+}
+
+Input* makeAdvanced(const Input* in)
```

Comment:
@jacobmerson I haven't seen an undefined behavior, using this approach. In any event, I will implement the `clone` just to be safe.

Reply:
added a copy c-tor here  https://github.com/SCOREC/core/pull/344/commits/11f56878f1264252c0cc7f7053dc306165fc180b and updated `makeAdvanced` to use that instead of const_cast